tgui/.eslintrc.yml

```diff
@@ -668,7 +668,7 @@ rules:
   # react/sort-prop-types: error
   ## Enforce the state initialization style to be either in a constructor or
   ## with a class property
-  react/state-in-constructor: error
+  # react/state-in-constructor: error
```

Comment:
This was necessary because even if `this.state` is made in constructor, TypeScript doesn't recognize it as being set...